docker/builder/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:20.04
+
+SHELL [ "/bin/bash", "-c" ]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Make sure we're starting with an up-to-date image
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get autoremove -y --purge && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+# To mark all installed packages as manually installed:
+#apt-mark showauto | xargs -r apt-mark manual
+
+RUN apt-get update && \
+    apt-get install -y \
+        apt-transport-https \
+        g++ \
+        gcc \
+        gnupg \
+        lsb-release \
+        lsof \
+        make \
+        ninja-build \
+        python3 \
+        python3-dev \
+        python3-pip \
+        python3-distro \
+        python3-jsonschema \
+        python3-packaging \
+        python3-psutil \
+        python3-pyodbc \
+        python3-requests \
+        sudo \
+        super \
+        wget \
+    && \
+    #pip3 --no-cache-dir install lief && \
```

Comment:
Probably don't need all of these packages, but that remains to be seen

Reply:
Same here - can optimize later?

Reply:
definitely